test/lib/flat_mutation_reader_assertions.hh

```diff
@@ -306,18 +306,23 @@ public:
         return *this;
     }
 
-    // If ck_ranges is passed, verifies only that information relevant for ck_ranges matches.
```

Comment:
Please adjust the changelog to explain what the omission is, I can't reverse-engineer it from the code.